extensions/wikia/MercuryApi/MercuryApiHelper.class.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+class MercuryApiHelper {
+
+	/**
+	 * Check if a string is a valid article title
+	 *
+	 * @param string $title
+	 * @return bool
+	 */
+	public static function isTitleStringValid( $title ) {
+		return !is_null( $title ) && trim( $title ) !== '';
```

Comment:
And by the way, those are valid links to the "The Elder Scrolls III: Morrowind" article content:
http://elderscrolls.wikia.com/wiki/%20The%20_%20Elder%20Scrolls%20III:%20Morrowind%20_%20
http://elderscrolls.wikia.com/wikia.php?controller=MercuryApi&method=getArticle&title=%20The%20_%20Elder%20Scrolls%20III:%20Morrowind%20_%20